src/main/scala/com/thenewmotion/akka/rabbitmq/ConnectionActor.scala

```diff
@@ -23,7 +23,7 @@ object ConnectionActor {
 
   def props(
     factory: ConnectionFactory,
-    reconnectionDelay: FiniteDuration = 10.seconds,
+    reconnectionDelay: FiniteDuration = 3.seconds,
```

Comment:
the default 10 seconds for reconnecting delay is a little bit too long. Do you think?